h/static/scripts/annotator/guest.coffee

```diff
@@ -44,8 +44,16 @@ module.exports = class Guest extends Annotator
   html: extend {}, Annotator::html,
     adder: '''
       <div class="annotator-adder">
-        <button class="h-icon-insert-comment" data-action="comment" title="New Note"></button>
-        <button class="h-icon-border-color" data-action="highlight" title="Highlight"></button>
+        <div class="adder-actions">
+          <div class="annotator-adder-action action-annotate">
+            <button id="adder-icon-annotate" class="h-icon-annotate" data-action="comment"></button>
+            <label for="adder-icon-annotate" class="adder-label">Annotate</label>
```

Comment:
OK, we're getting close :)

In discussion on Slack, Nick just mentioned an issue where there is a region of what looks like the button that is between the `<button>` and the `<label>`. Clicking that doesn't create a new annotation. I think the right way to address this is going to be to put the label just as text inside the button. eg. `<button ...><span class="h-icon-annotate"...></span>Annotate</button>`, then use `display: flex; flex-direction: column` to layout the contents of the button.